pkg/authenticator/ecrsecret_test.go

```diff
@@ -0,0 +1,243 @@
+package authenticator
+
+import (
+	"context"
+	"encoding/base64"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/fake"
+	"k8s.io/client-go/rest"
+
+	api "github.com/aws/eks-anywhere-packages/api/v1alpha1"
+)
+
+func TestAuthFilename(t *testing.T) {
+	config := rest.Config{}
+	t.Run("golden path for set HELM_REGISTRY_CONFIG", func(t *testing.T) {
+		testfile := "/test.txt"
+		t.Setenv("HELM_REGISTRY_CONFIG", testfile)
+		ecrAuth, _ := NewECRSecret(&config)
```

Comment:
You don't get to skip checking errors in test files either! 😄 
```suggestion
		ecrAuth, err := NewECRSecret(&config)
		require.NoError(t, err)
```